docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -1,6 +1,11 @@
 = Quickstart
+:navtitle: Quickstart
+:page-layout: tutorial
+:page-icon-role: bg-[var(--ds-neutral-900)]
+:page-toclevels: 1
+:keywords: Machine Learning Frameworks, Embedding Services, Data Warehouses, SDK
```

Comment:
is this our standard set? Should it have some generative ai words as well?

Reply:
It is the standard set